packages/eslint-plugin-pf-codemods/lib/rules/v5/backgroundImageSrcMap-remove-interface.js

```diff
@@ -0,0 +1,63 @@
+const { getPackageImports } = require("../../helpers");
+
+// https://github.com/patternfly/patternfly-react/pull/8931
+module.exports = {
+  meta: { fixable: "code" },
+  create: function (context) {
+    const backgroundImgSrcMapImport = getPackageImports(
+      context,
+      "@patternfly/react-core"
+    ).find((specifier) => specifier.imported.name == "BackgroundImageSrcMap");
+
+    return !backgroundImgSrcMapImport
+      ? {}
+      : {
+          ImportDeclaration(node) {
+            const backgroundImgSrcMapSpecifier = node.specifiers.find(
+              (specifier) =>
+                specifier.local.name === backgroundImgSrcMapImport.local.name
+            );
+
+            if (backgroundImgSrcMapSpecifier) {
+              context.report({
+                node,
+                message:
+                  "The BackgroundImageSrcMap interface has been removed.",
+                fix(fixer) {
+                  const tokenAfter = context
+                    .getSourceCode()
+                    .getTokenAfter(backgroundImgSrcMapSpecifier).value;
+
+                  const { range } = backgroundImgSrcMapSpecifier;
+                  return node.specifiers.length > 1
+                    ? fixer.removeRange([
+                        range[0],
+                        tokenAfter === "," ? range[1] + 1 : range[1],
```

Comment:
this will most likely be fine (people probably won't do `import { foo , bar }` with the space)... but we could just the token after rather than the value of it, check the value of it.. then use takenAfter.range[1] if it's a comma